docs/fundamentals/service-defaults.md

```diff
@@ -111,6 +117,125 @@ The `MapDefaultEndpoints` method:
 
 For more information, see [.NET Aspire health checks](health-checks.md).
 
+## Custom service defaults
+
+If the default service configuration provided by the project template is not sufficient for your needs, you have the option to create your own service defaults project. This is especially useful when your consuming project, such as a Worker project or WinForms project, cannot or does not want to have a `FrameworkReference` dependency on `Microsoft.AspNetCore.App`.
+
+To do this, create a new .NET 8.0 class library project and add the necessary dependencies to the project file, consider the following example:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Hosting" />
+    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" />
+    <PackageReference Include="Microsoft.Extensions.Http.Resilience" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" />
+    <PackageReference Include="OpenTelemetry.Instrumentation.GrpcNetClient" />
```

Comment:
Note that this package isn't stable yet, which is why we removed it from our templates. We may want to remove it from here as well.

Reply:
Yes, I'm going to do another pass on updating that service-defaults content. Will fix this.